### PR TITLE
(src updates) pawread

### PR DESCRIPTION
**Pull request by K1ngfish3r**
hermes bug
novelupdates search update
ranobes (no filter yet)

### CODE REVIEW

File: plugins/english/pawread.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+import { CheerioAPI, load as parseHTML } from "cheerio";
 import { fetchApi, fetchFile } from "@libs/fetch";
 import { Plugin } from "@typings/plugin";
 import { Filters, FilterTypes } from "@libs/filterInputs";
 
 class PawRead implements Plugin.PluginBase {
     id = "pawread";
     name = "PawRead";
-    version = "1.1.0";
+    version = "2.1.0";
```
Comment (nyagami) [inline plugins/english/pawread.ts:9]:
```suggestion
    version = "2.0.0";
```

File: plugins/english/novelupdates.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { Plugin } from "@typings/plugin";
 class NovelUpdates implements Plugin.PluginBase {
     id = "novelupdates";
     name = "Novel Updates";
-    version = "0.5.0";
+    version = "0.6.0";
```
Comment (nyagami) [inline plugins/english/novelupdates.ts:9]:
```suggestion
    version = "0.5.1";
```

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "master"
+    ]
+}
```
Comment (nyagami) [inline .vscode/settings.json:5]: what is this

Comment (K1ngfish3r) [reply .vscode/settings.json:5]: oh, i was using github.dev and it gave me this error message, lemme remove that

**Review by nyagami — APPROVED** [review]

**Pull request merged**